metadatafields.md

```diff
@@ -71,29 +71,47 @@ Return codes:
 
 To create a metadata field, perform a post with the JSON below when logged in as admin.
 
-```
+```json
 {
   "element": "contributor",
   "qualifier": "tester",
-  "scopeNote": "An agent which provided illustrations for the resource",
-  "type": "metadatafield"
+  "scopeNote": "An agent which provided illustrations for the resource"
 }
 ```
 
+Return codes:
+* **201 Created** - if the operation succeed
+* **400 Bad Request** - if the body of the request can't be parsed (for example because of a missing field)
+* **401 Unauthorized** - if you are not authenticated
+* **403 Forbidden** - if you are not logged in with sufficient permissions
+* **422 Unprocessable Entity** - if the request is well-formed, but is invalid based on the given data.
+  * If the schema ID does not exist
+  * If you attempt to create a field with an empty `element` or if it contains dots.
+  * If you attempt to create a field with a `qualifier` contains dots.
```

Comment:
For both element & qualifier, again we could just say "invalid characters".  Or, we could list them all.  In the implementation PRs, I suggested extending the list (for both) to "dots, commas or spaces."